ui/src/stores/flow.js

```diff
@@ -675,6 +675,7 @@ export default {
             state.tasksWithMetrics = tasksWithMetrics
         },
         setFlowYaml(state, flowYaml) {
+            console.log("setFlowYaml", 1)
```

Comment:
Just remove this log and we're good to go! 🚀

Reply:
Removing log
```suggestion
```